pulitzer.gemspec

```diff
@@ -32,6 +32,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sqlite3', '~> 1.3'
   s.add_development_dependency 'minitest', '~> 5.1'
   s.add_development_dependency 'byebug', '~> 8.0'
+  s.add_development_dependency 'pry-rails', '~> 0.3', '< 1.0'
```

Comment:
I added it because is the default debug tool for the last rails versions, but I think you are right when you say that we already now pry and for consistency cross the projects. :+1: